integrations/vault/auths/approle/src/main/java/io/helidon/integrations/vault/auths/approle/AppRoleVaultAuth.java

```diff
@@ -85,55 +88,57 @@ public Optional<RestApi> authenticate(Config config, Vault.Builder vaultBuilder)
-                .address(address)
-                .disableVaultAuthDiscovery()
-                .updateWebClient(vaultBuilder.webClientUpdater())
-                .faultTolerance(vaultBuilder.ftHandler())
-                .addVaultAuth(NoVaultAuth.create());
+                                               .address(address)
+                                               .disableVaultAuthDiscovery()
+                                               .updateWebClient(vaultBuilder.webClientUpdater())
+                                               .faultTolerance(vaultBuilder.ftHandler())
+                                               .addVaultAuth(NoVaultAuth.create());
 
-        vaultBuilder.baseNamespace().ifPresent(loginVaultBuilder::baseNamespace);
+        vaultBuilder.baseNamespace()
+                    .ifPresent(loginVaultBuilder::baseNamespace);
 
         Vault loginVault = loginVaultBuilder.build();
         String methodPath = Optional.ofNullable(this.methodPath)
-                .orElseGet(() -> config.get("auth.app-role.path")
-                        .asString()
-                        .orElse(AppRoleAuthRx.AUTH_METHOD.defaultPath()));
+                                    .orElseGet(() -> config.get("auth.app-role.path")
+                                                           .asString()
+                                                           .orElse(AppRoleAuth.AUTH_METHOD.defaultPath()));
 
-        LOGGER.log(Level.INFO, "Authenticated Vault " + address + "/" + methodPath + " using AppRole, roleId \"" + appRoleId
-                + "\"");
+        LOGGER.log(Level.INFO,
+                "Authenticated Vault {0}/{1} using AppRole, roleId \"{2}\"",
+                address, methodPath, appRoleId);
 
         return Optional.of(AppRoleRestApi.appRoleBuilder()
-                                   .webClientBuilder(webclient -> {
-                                       webclient.baseUri(address + "/v1");
-                                       vaultBuilder.baseNamespace()
-                                               .ifPresent(ns -> webclient.addHeader("X-Vault-Namespace", ns));
-                                       vaultBuilder.webClientUpdater().accept(webclient);
-                                   })
-                                   .faultTolerance(vaultBuilder.ftHandler())
-                                   .auth(loginVault.auth(AppRoleAuthRx.AUTH_METHOD, methodPath))
-                                   .appRoleId(appRoleId)
-                                   .secretId(secretId)
-                                   .build());
+                                         .webClientBuilder(webclient -> {
+                                             webclient.baseUri(address + "/v1");
+                                             vaultBuilder.baseNamespace()
+                                                         .ifPresent(ns -> webclient.header(Http.Header.create("X-Vault-Namespace"), ns));
```

Comment:
Same comment as above, we should overload the `header` method to accept a string for the header name.

Reply:
Again, please create a constant `HeaderName` for namespace and use it.

Reply:
Updated.